style.css

```diff
@@ -95,3 +93,279 @@ h2 {
 .social-link-item {
   margin-right: 12px;
 }
+
+.react-work {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  height: fit-content;
+}
+
```

Comment:
## Changes Required
According to project requirements, the main container of the projects has a `display:grid`